node_modules/oae-admin/manageuser/manageuser.html

```diff
@@ -192,6 +192,16 @@ <h3 id="manageuser-editprofile-header-container"><!-- --></h3>
                         </div>
                     </label>
                 </div>
+                <div class="col-sm-3 text-center">
+                    <label for="manageuser-email-never" {if userProfile.emailPreference === 'never'} class="checked"{/if}>
+                        <i class="fa fa-check hide"></i>
+                        <div class="oae-large-options well">
+                            <input type="radio" id="manageuser-email-never" value="never" name="manageuser-email-group" class="pull-left" tabindex="0" {if userProfile.emailPreference === 'never'} checked{/if}/>
+                            <i class="fa fa-ban large text-center {if userProfile.emailPreference === 'weekly'} selected{/if}"></i>
```

Comment:
Interestingly enough, I don't see a visual difference between applying `selected` to the icon and not applying it. Can you try to find if it still does anything anywhere and if not, can you go ahead and remove the `if` statement in all places where we use the `oae-large-options` ?